tests/unit/worker/worker_test.py

```diff
@@ -178,64 +163,63 @@ def test_add_constructor_after_decorator():
     assert decorator in zeebe_worker._after
 
 
-def test_create_before_decorator_runner():
+def test_create_before_decorator_runner(zeebe_worker, task):
     task.before(decorator)
     job = random_job(task=task)
     job.variables = {"x": str(uuid4())}
     decorators = zeebe_worker._create_before_decorator_runner(task)
     assert isinstance(decorators(job), Job)
 
 
-def test_handle_one_job():
+def test_handle_one_job(zeebe_worker, task):
     job = random_job(task=task)
 
     with patch("pyzeebe.worker.worker.ZeebeWorker._get_jobs") as get_jobs_mock:
```

Comment:
It's possible, because you're creating `zeebe_worker` anew on every test function. So each function can "destroy" `zeebe_worker` as much as needed.

_This is contrary to what c1f31c5c6fd9f25a6a25383268c93314f873303c commit states_:

> Only grpc fixtures require the module scope, other fixtures will just retain their original value (not refreshed every test)

Ref. [pytest docs](https://docs.pytest.org/en/stable/fixture.html#scope-sharing-fixtures-across-classes-modules-packages-or-session):
> the default is to invoke once per test function